### ISSUE TITLE
dayjs function toArray() produces an illogical result

### ISSUE THREAD

**Issue by elementalPRESS** [iamkun/dayjs#1847]
The `toArray()` function is described in the documentation:

	dayjs('2019-01-25').toArray() // [ 2019, 0, 25, 0, 0, 0, 0 ]

What I don't understand: why is the month value always `-1`. In the example above I would have expected `1` as the value for the month. That seems absolutely illogical to me and it would be nice to get an explanation for this.

**Comment by neospirit** [comment]
I agree with @elementalPRESS and so, meanwhile for a potential fix or explanation, is the result consistent on all months too?

Also, would it be safe to add +1 to the month digit to get the real result we want?
